common/models/Moods.ts

```diff
@@ -29,8 +29,25 @@ export namespace Moods {
 
     export const Step = 10;
 
-    export function getTitle(mood: Moods) {
-        return Helper.value.valueToString(findNearest(mood));
+    export function getTitle(mood: Moods, isSleep: boolean = false) {
+        const str = Helper.value.valueToString(findNearest(mood));
+        if (isSleep) {
+            switch (str) {
+                case 'very bad':
+                    return 'not at all';
+                case 'bad':
+                    return 'not really';
+                case 'ok':
+                    return 'mostly';
+                case 'good':
+                    return 'pretty good';
+                case 'very good':
+                    return 'yes';
```

Comment:
Laura is working to finalize these labels, but we can keep them like this for now.